.gitignore

```diff
@@ -1,2 +1,3 @@
 node_modules
-.env.*
\ No newline at end of file
+.env.test
```

Comment:
.env.* was targeting all .env files here, therefore there was no need to remove it or to add .env.test & .env.development